### ISSUE TITLE
Documentation: Docker examples could use a GUI one

### ISSUE THREAD

**Issue by l0b0** [NixOS/nixpkgs#293979]
## Problem

I'm trying to write a simple .nix file to create a container image which runs a GUI application, ideally as a non-root user. There's various snippets and hints around, but no solid, simple example like [the ones in this repo](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/docker/examples.nix).

## Proposal

Create a Docker example which runs a small GUI application as a proof of concept.

## Checklist

- [x] checked [latest Nixpkgs manual] \([source][nixpkgs-source]) and [latest NixOS manual] \([source][nixos-source])
- [x] checked [open documentation issues] for possible duplicates
- [x] checked [open documentation pull requests] for possible solutions

[latest Nixpkgs manual]: https://nixos.org/manual/nixpkgs/unstable/
[latest NixOS manual]: https://nixos.org/manual/nixos/unstable/
[nixpkgs-source]: https://github.com/NixOS/nixpkgs/tree/master/doc
[nixos-source]: https://github.com/NixOS/nixpkgs/tree/master/nixos/doc/manual
[open documentation issues]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+label%3A%229.needs%3A+documentation%22
[open documentation pull requests]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+is%3Apr+label%3A%228.has%3A+documentation%22%2C%226.topic%3A+documentation%22

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by eclairevoyant** [comment]
I don't think this is something that fits well in docs, it's a specific usecase. Wiki would be a better fit.
